c-pallets/segment-book/src/lib.rs

```diff
@@ -974,6 +989,7 @@ pub mod pallet {
 					Err(Error::<T>::FileTypeError)?;
 				},
 			}
+			// This condition wasn't checked already returning Ok(())?
```

Comment:
Yes, because the miner does not exist, it is a normal situation. When the miner is offline, no other processing will be performed, so `OK (())` will be returned